src/release/whats-new.md

```diff
@@ -22,6 +25,50 @@ and review the [Dart changelog][].
 [Dart Announce]: {{site.groups}}/a/dartlang.org/g/announce
 [Dart changelog]: {{site.github}}/dart-lang/sdk/blob/main/CHANGELOG.md
 
+## 15 February 2024: Valentine's-Day-adjacent 3.19 release
+
+Flutter 3.19 is live! For more information,
+check out the [Flutter 3.19 umbrella blog post][3.19-umbrella]
+and the [Flutter 3.19 technical blog post][3.19-tech].
+
+You might also check out the [Dart 3.3 release][] blog post.
+
+[3.19-tech]: {{site.flutter-medium}}/whats-new-in-flutter-3-19-58b1aae242d2
+[3.19-umbrella]: {{site.flutter-medium}}/starting-2024-strong-with-flutter-and-dart-cae9845264fe
+[Dart 3.3 release]: {{site.medium}}/dartlang/new-in-dart-3-3-extension-types-javascript-interop-and-more-325bf2bf6c13
+
+**Docs updated or added since the 3.16 release**
+
+* A new page on [migrating from Material 2 to Material 3][]
```

Comment:
This is missing a link definition. I guess you mean https://docs.flutter.dev/release/breaking-changes/material-3-migration?

Reply:
I fixed that a few minutes ago?